conformance/conformance_php.php

```diff
@@ -29,7 +30,10 @@ function doTest($request)
     $test_message = new \Protobuf_test_messages\Proto3\TestAllTypesProto3();
     $response = new \Conformance\ConformanceResponse();
     if ($request->getPayload() == "protobuf_payload") {
-      if ($request->getMessageType() == "protobuf_test_messages.proto3.TestAllTypesProto3") {
+      if ($request->getMessageType() == "conformance.FailureSet") {
```

Comment:
Do you still need this?

Reply:
I think so, Gerben added a new test in RunSuite (see the change in conformance/conformance_test.cc), this test has message type with "conformance.FailureSet".

And I just realize it seems we don't need to add the require for this message.